UI/window-basic-status-bar.hpp

```diff
@@ -6,6 +6,17 @@
 #include <util/platform.h>
 #include <obs.h>
 
+#define OPT_DYN_BITRATE "DynamicBitrate"
+
+enum dynamicBitrateState {
+	BITRATE_EQUAL_INITIAL_BITRATE,
+	BITRATE_SWITCHING_DOWN,
+	BITRATE_SWITCHING_STATIONARY,//means bitrate is lower than initial bitrate so that
+								// the bitrate switching mechanism is still in place
```

Comment:
These comments seem excessively indented?

Reply:
you probably reviewed from github ? github display seems faulty; in VS2017 ide and notepad++ the  lines are aligned

Reply:
I always review from GitHub, initially.  In Notepad++, with a tab-size of 8 (this project's setting), it looks just like GitHub's representation.
![image](https://user-images.githubusercontent.com/624931/39700202-601769f0-51ca-11e8-9c1a-46d3ade18b3c.png)

In VS2017:
![image](https://user-images.githubusercontent.com/624931/39700326-c32cb05e-51ca-11e8-8928-98493ee6543a.png)

Reply:
oh i guess i have tabs at 4 then ... shit, lemme fix that